### ISSUE TITLE
Images should always have alt text

### ISSUE THREAD

**Issue by emilypstewart** [j4502/Local-Elections#10]
Ran your source on your revised [homepage](https://github.com/j4502/Local-Elections/blob/develop/index.html) through the validator and found a couple of small errors.

![Screenshot of W3C Validator errors](https://cloud.githubusercontent.com/assets/774386/11022012/3c128adc-8619-11e5-9c1a-3c7a079d1295.png)

Good news: they're super easy fixes.

Just need to add alt text to some of the images. 

**Incorrect examples:**
`<img src="/assets/images/my-image.png" width="200" alt="">`

 :-1: 

`<img src="/assets/images/my-image.png" width="200" alt="photo">`

:unamused: 
`<img src="/assets/images/my-image.png" width="200" alt=" ">`

:no_good:  
`<img src="/assets/images/my-image.png" width="200">`

**Correct**
`<img src="/assets/images/my-image.png" width="200" alt="aerial photo of Seattle at night">`

---
### What is alt text?

Alt text is used by screen readers (technology that helps people with visual impairments interact with your website). It uses the text to describe the image to a user relying on that technology. It should be concise, but still descriptive enough to give context.

**Example:**

![photo of businessman snorkeling underwater while casually using tablet](https://cloud.githubusercontent.com/assets/774386/11022076/4f7058b4-861b-11e5-87d7-657f59cb4356.jpg)

for this photo you could say `alt="photo of businessman snorkeling underwater while casually using tablet"` or something.

**Comment by emilypstewart** [comment]
(Good job adding alt text to the first photo of the flag, though!)
